### PR TITLE
Check IP Services - Plugin

### PR DESCRIPTION
**Pull request by NOYB**
Configurable check IP services.
Used by Dynamic DNS services to check IP addresses.
Configure as many check IP services as desired.  The first (highest in the list) that is enabled will be used as the default.
Include verify peer option.
A future possibility could be to add an option to the Dynamic DNS entires to use a specific check IP service.
Allows additional/private check IP services to be added and set as the default.

### CODE REVIEW

File: src/www/services_checkip.php [hunk 1]
```diff
@@ -0,0 +1,192 @@
+<?php
+
+/*
```
Comment (fichtner) [inline src/www/services_checkip.php]: No Copyrights for you?

Comment (NOYB) [reply src/www/services_checkip.php]: Only thing I'm interested in is retaining my right to use personally or in other projects.
As far as anyone else using it.  Far as I'm concerned my portions of the code is public domain.
If necessary for that I guess you could add a NOYB at Github copyright.

**Comment by fichtner** [comment]
Ok, fine, since this is BSD license your right to use is unrestricted so no need to do anything.

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**